srcpkgs/libgdal/template

```diff
@@ -46,3 +82,17 @@ libgdal-devel_package() {
 		vmove usr/lib/*.so
 	}
 }
+
+python3-gdal_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
+	short_desc+=" - Python3 bindings"
+	pkg_install() {
+		vmove usr/bin/*.py
+		vmove "usr/lib/python*"
+		vlicense $wrksrc/$build_wrksrc/LICENSE.TXT
+		vdoc $wrksrc/$build_wrksrc/swig/python/README.rst
+		mkdir -p ${PKGDESTDIR}/usr/share/${pkgname}
+		cp -r $wrksrc/$build_wrksrc/swig/python/samples ${PKGDESTDIR}/usr/share/${pkgname}/examples
```

Comment:
`$wrksrc/$build_wrksrc` should be cwd, so no need to add it to the path (for all the above)

Reply:
```suggestion
		vcopy swig/python/samples usr/share/python3-gdal/examples
```